### PR TITLE
New package: astroid-0.13

### PR DESCRIPTION
**Pull request by bitmeise**

### CODE REVIEW

**Comment by cr6git** [comment]
have a look at https://github.com/voidlinux/void-packages/pull/14794

**Comment by bitmeise** [comment]
Thanks for the reference. I forgot to have a look at the old github. Please bear with me, as this is my first public contribution to void.

Although build time is a lot longer without ninja, I really like the clean template.

File: srcpkgs/astroid/template [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Template file for 'astroid'
+pkgname=astroid
```
Comment (maxice8) [inline srcpkgs/astroid/template]: version= directly below pkgname=
revision= directly below version=

build_style= directly below revision=
hostmakedepends= directly below build_style=
makedepends= directly below hostmakedepends=, cut the line at 80 columns and the new line should have a whitespace prefix

**Comment by bitmeise** [comment]
done

File: srcpkgs/astroid/template [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+revision=1
+build_style=cmake
+hostmakedepends="cmake pkg-config"
+makedepends="libnotmuch-devel gtkmm-devel webkit2gtk-devel libsass-devel
+ gmime-devel boost-devel protobuf-devel vte3-devel gobject-introspection
+ libpeas-devel"
+short_desc="Graphical threads-with-tags style email client for Notmuch"
+homepage="http://astroidmail.github.io/"
+license="GPL-3.0-or-later"
+maintainer="Ameise <ameise@bitparlament.de>"
```
Comment (maxice8) [inline srcpkgs/astroid/template]: maintainer= directly below short_desc=
homepage= directly above distfiles=

File: srcpkgs/astroid/template [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+distfiles="https://github.com/astroidmail/astroid/archive/v${version}.tar.gz"
+checksum="52a5e34b1d8dc597d7fdc9d63d258e519e6d7c3324110de737e5a62906256fb5"
```
Comment (maxice8) [inline srcpkgs/astroid/template]: checksum doesn't need to be quoted

**Comment by bitmeise** [comment]
okay

File: srcpkgs/astroid/template [hunk 4]
```diff
@@ -0,0 +1,15 @@
+# Template file for 'astroid'
+pkgname=astroid
+version=0.13
+revision=1
+build_style=cmake
+hostmakedepends="cmake pkg-config"
```
Comment (cr6git) [inline srcpkgs/astroid/template]: remove cmake from the hostmakedepends

**Comment by bitmeise** [comment]
You're right. That's a leftover of the other build style. Removed.

**Pull request merged**